packages/dashboard/app/components/ChatView.css

```
@@ -5,6 +5,9 @@
   flex: 1;
   width: 100%;
```

Stylelint (17.12.0)

[error] Duplicate property "width" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)